examples/pixi-lights/index.ts

```diff
@@ -0,0 +1,113 @@
+import * as PIXI from 'pixi.js';
+
+import { AmbientLight, DirectionalLight, PointLight, diffuseGroup, normalGroup } from 'pixi-lights';
+import { AppLoaderPlugin, Shadow } from '../../src';
+import { Application, Container, InteractionEvent, Texture } from 'pixi.js';
+
+// Initialise the shadows plugin
+Application.registerPlugin(AppLoaderPlugin);
+
+/* The actual demo code: */
+
+// Create your application
+const width = 800;
+const height = 500;
+const app = new PIXI.Application({ width, height });
```

Comment:
```suggestion
const app = new PIXI.Application({ width, height, fov: { useLights: true } });
```

You forgot to indicate to use pixi-lights here.

Reply:
Thanks!